executor/executor.go

```diff
@@ -879,6 +921,8 @@ func (e *HashSemiJoinExec) Fields() []*ast.ResultField {
 	return nil
 }
 
+// prepare runs at the first time 'Next' is called, it reads all data from small table and stores
```

Comment:
Prepare runs the first time when 'Next' is called and it reads all data from the small table and stores